src/Acuminator/Acuminator.Analyzers/Resources.resx

```diff
@@ -684,4 +681,10 @@ public virtual int? SomeDacField{ get; set; }</value>
   <data name="PX1007FixAddInheritdocTag" xml:space="preserve">
     <value>Add the &lt;inheritdoc/&gt; tag for the projection DAC property with `cref` attribute referencing the mapped original DAC property</value>
   </data>
+  <data name="PX1041Title" xml:space="preserve">
```

Comment:
@EAndrosova  please review the messages for the PX1041 diagnostic and its code fix